src/EnergyPlus/HVACDuct.cc

```diff
@@ -251,62 +249,6 @@ namespace HVACDuct {
         }
     }
 
-    void InitDuct(EnergyPlusData &state, int const DuctNum) // number of the current duct being simulated
-    {
-
-        // SUBROUTINE INFORMATION:
-        //       AUTHOR         Fred Buhl
-        //       DATE WRITTEN   17May2005
-        //       MODIFIED       na
-        //       RE-ENGINEERED  na
-
-        // PURPOSE OF THIS SUBROUTINE:
-        // This subroutine is for initializations of the Duct Components
-
-        // METHODOLOGY EMPLOYED:
-        // Uses the status flags to trigger initializations
-
-        // REFERENCES:
-        // na
-
-        // USE STATEMENTS:
-        // na
-
-        // Locals
-        // SUBROUTINE ARGUMENT DEFINITIONS:
-
-        // SUBROUTINE PARAMETER DEFINITIONS:
-        // na
-
-        // INTERFACE BLOCK SPECIFICATIONS
-        // na
-
-        // DERIVED TYPE DEFINITIONS
-        // na
-
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        bool MyOneTimeFlag(true);
-        Array1D_bool MyEnvrnFlag;
-
-        // do one time initializations
-        if (MyOneTimeFlag) {
-            // initialize the environment and sizing flags
-            MyEnvrnFlag.dimension(state.dataHVACDuct->NumDucts, true);
-
-            MyOneTimeFlag = false;
```

Comment:
[src/EnergyPlus/HVACDuct.cc:296]:(style),[unreadVariable],Variable 'MyOneTimeFlag' is assigned a value that is never used.